remoting/subsystem/src/main/java/org/jboss/as/remoting/AbstractStreamServerService.java

```diff
@@ -94,8 +91,8 @@ public void start(final StartContext context) throws StartException {
                 builder.addAll(connectorPropertiesOptionMap);
             }
             OptionMap resultingMap = builder.getMap();
-            if (log.isTraceEnabled()) {
-                log.tracef("Resulting OptionMap %s", resultingMap.toString());
+            if (RemotingLogger.ROOT_LOGGER.isTraceEnabled()) {
+                RemotingLogger.ROOT_LOGGER.tracef("Resulting OptionMap %s", resultingMap.toString());
             }
```

Comment:
Don't worry about this unless you make some other change, but this can be changed to one line:

RemotingLogger.ROOT_LOGGER.tracef("Resulting OptionMap %s", resultingMap);